spec/Classes.tex

```diff
@@ -307,12 +370,12 @@ \subsection{Class Values}
 \end{chapelcompopts}
 \begin{chapeloutput}
 \end{chapeloutput}
-When the variable \chpl{c} is declared, it initially has the value
-of \chpl{nil}.  The next statement assigned to it an instance of the
-class \chpl{C}.  The declaration of variable \chpl{c2} shows that these steps can
-be combined.  The type of \chpl{c2} is also \chpl{C}, determined implicitly from
-the the initialization expression.  Finally, an object of type \chpl{D} is created and
-assigned to \chpl{c}.
+When the variable \chpl{c} is declared, it initially has the value of
+\chpl{nil}.  The next statement assigned to it an instance of the class
+\chpl{C}.  The declaration of variable \chpl{c2} shows that these steps
+can be combined.  The type of \chpl{c2} is also \chpl{C}, determined
+implicitly from the the initialization expression.  Finally, an object
+of type \chpl{D} is created and assigned to \chpl{c}.
```

Comment:
This example lacks clarity about nilability of `c2`. Depending on the behavior of `.borrow()` - see my earlier comment - need to add or edit that:
* the type of `c2` is nilable `is also \chpl{C?}`, or
* the above code would produce an error upon `c.borrow()` if it were in a non-prototype module.

Oh, either way, the type of `c2` is actually `borrowed`, so need to remove the `also`.